test/test_retryable_writes.py

```diff
@@ -453,6 +454,28 @@ def test_batch_splitting_retry_fails(self):
             self.assertEqual(final_txn, expected_txn)
         self.assertEqual(coll.find_one(projection={'_id': True}), {'_id': 1})
 
+    @client_context.require_version_min(4, 4)
+    @client_context.require_failCommand_fail_point
+    def test_retryable_write_error_label_is_propagated(self):
+        listener = OvertCommandListener()
+        client = rs_or_single_client(
+            retryWrites=True, event_listeners=[listener])
+
+        fail_insert = {
+            'configureFailPoint': 'failCommand',
+            'mode': {'times': 2},
+            'data': {
+                'failCommands': ['insert'],
+                'writeConcernError': {
+                    'code': 91,
+                    'errmsg': 'Replication is being shut down'}}}
+
+        with self.fail_point(fail_insert):
+            with self.assertRaises(WriteConcernError) as cm:
+                client.testdb.testcoll.insert_one({})
```

Comment:
I suggest using the standard `pymongo_test`  database name here.

Reply:
Done.